tests/output/JSONEncoderTest.py

```diff
@@ -75,6 +76,11 @@ def test_time(self):
         self.assertEquals('"' + tf.isoformat() + '"',
                           json.dumps(tf, **self.kw))
 
+    def test_re_object(self):
+        uut = re.compile('x')
+        self.assertEqual('"' + uut.pattern + '"',
+                         json.dumps(uut, **self.kw))
```

Comment:
yeeeees our jsonencoder is a wonder thing meant to deal with about anything you throw at it :)